Packs/RSS/pack_metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "RSS",
+    "description": "Simple RSS reader, imports new posts as Incidents.  All rss fields (e.g. author, published, tags) are available.",
```

Comment:
Please try to write a more indicative description as using this people can know if they would want to use your pack.